src/pd/Client.cc

```diff
@@ -75,9 +75,11 @@ Client::~Client()
 
 void Client::update(const std::vector<std::string> & addrs, const ClusterConfig & config_)
 {
-    uninit();
-
-    init(addrs, config_);
+    urls = addrsToUrls(addrs, config_);
+    config = config_;
```

Comment:
Make sure write to `urls` and `config` are thread safe.

Reply:
Because the update method only called in `tiflash.ConfigReloader`, and the config reloader only update once per 2 seconds, so I didn't add a mutex.